src/main/java/com/google/codeu/servlets/HomeServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.codeu.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This servlet prints out the HTML for the homepage. You wouldn't do this in a real codebase,
+ * but this is meant to demonstrate getting a Blobstore URL and using it in a form to allow
+ * a user to upload a file.
+ */
+@WebServlet("/home")
```

Comment:
i imagine this is only testing right now, yea? can we move this to something like /upload  or do you plan to delete this servlet at some point?